app/assets/stylesheets/shared/form.scss

```diff
@@ -39,117 +39,132 @@
   }
 }
 
-// TODO: these don't really with with slimselect v2
+// These styles work with Slim Select v2, not tested with v1
 .ss-main {
   &.form-select {
-    padding: 0;
+    padding: 0.75rem 0.75rem;
     min-height: $input-height;
     height: auto;
   }
 
-  .ss-single-selected,
-  .ss-multi-selected {
-    padding: $input-btn-padding-y $input-btn-padding-x;
-    line-height: $line-height-base;
-    border: 0;
-    border-radius: $input-border-radius;
-    color: $input-color;
-
-    &.ss-disabled {
-      background: $input-disabled-bg;
-      border-color: $input-disabled-border-color;
-      cursor: default;
+  // selection colors
+  .ss-values .ss-value {
+    animation: none;
+    background-color: $primary;
+    margin: 2px 5px 0 0;
+
+    .ss-value-delete,
+    .ss-value-delete svg {
+      width: 16px;
+      height: 16px;
+      border-left: none;
```

Comment:
Note that previously everything was wrapped in the `.ss-main` selector. Its still a selector, but doesn't apply to everything anymore. 

So the diffs won't line up perfectly here.